test/integration/run-bats.sh

```diff
@@ -15,7 +15,7 @@ function quiet_run () {
 
 function cleanup_machines() {
     if [[ $(machine ls -q | wc -l) -ne 0 ]]; then
-        quiet_run machine rm -f $(machine ls -q)
+        quiet_run machine rm -y -f $(machine ls -q)
```

Comment:
If you're already forcing the removal `-f` do you also need to specify an automatic yes `-y` ?